portfolio/src/main/webapp/style.css

```diff
@@ -316,3 +316,65 @@ section .courses ul li span {
   margin: 5px;
 }
 
+/* -------------- Project Hover Cards --------------*/
+
+.container {
+  position: relative;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items:center;
+  flex-wrap: wrap;
+  padding: 30px;
+}
+
+.container .card {
+  max-width: 300px;
```

Comment:
IMO, 300px makes it a little hard to read (each line is too short so there's a lot of short lines)